packages/cadl-ranch-specs/http/type/model/inheritance/multiple-discriminator/main.tsp

```diff
@@ -0,0 +1,222 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Illustrates multiple level inheritance with multiple discriminators.")
+@scenarioService("/type/model/inheritance/multiple-discriminator")
+namespace Type.Model.Inheritance.MultipleDiscriminator;
+
+@doc("This is base model for polymorphic multiple levels inheritance with a discriminator.")
+@discriminator("kind")
+model Fish {
+  age: int32;
+}
+
+@doc("The second level model in polymorphic multiple levels inheritance and it defines a new discriminator.")
+@discriminator("sharktype")
+model Shark extends Fish {
+  kind: "shark";
```

Comment:
```suggestion
  kind: "shark";
  sarktype: string; 
```
recommended way is to still provide the base class with the discirminator property. Omitting it is a legacy feature from when we didn't allow property override

Reply:
maybe we can add a separate test for the 2 syntax in the single-discriminator

Reply:
done